pkg/server/endpoints/node/handler.go

```diff
@@ -712,7 +712,12 @@ func (h *Handler) signCSRs(ctx context.Context,
 			if err != nil {
 				return nil, err
 			}
-			svid, err := h.buildCASVID(ctx, csr, e.Ttl)
+			svid, err := h.buildCASVID(ctx, csr,
+				ca.X509Params{
+					TTL:     time.Duration(e.Ttl) * time.Second,
+					DNSList: e.DnsList,
```

Comment:
non-blocking comment: Maybe take belt-and-suspenders approach here, and not pass dns since we know we're not supposed to be setting it?